src/test/components/ProfileCallTreeView.test.js

```diff
@@ -20,7 +20,7 @@ import { autoMockCanvasContext } from '../fixtures/mocks/canvas-context';
 import { storeWithProfile } from '../fixtures/stores';
 import {
   getBoundingBox,
-  createSelectChanger,
+  changeSelect,
```

Comment:
I changed this tool: indeed previously we were taking the render result, but now we don't need it anymore thanks to `screen`, so I simplified this all.